### DISCUSSION TITLE
Yolov5_s6 Release 6.2

### DISCUSSION THREAD

**Discussion by Kanan99** [obss/sahi#1027] [Q&A] [answered]
Hey,

 I've been exploring the SAHI repository and I'm impressed by its capabilities. I'm particularly interested in incorporating my own custom-trained model into the framework.

I've been working with the YOLOv5 model, specifically version 6.2 from the Ultralytics [repository](https://github.com/ultralytics/yolov5/tree/v6.2). My goal is to integrate this custom model with SAHI for a specific application.

Before proceeding, I wanted to reach out to the community to inquire about any guidelines, best practices, or potential compatibility concerns I should be aware of when integrating a custom-trained model with SAHI. Additionally, I'm open to any advice or recommendations from those who have experience with similar integrations.

Any insights or pointers you could provide would be greatly appreciated. Thank you in advance for your time and assistance.

**Comment by fcakyon** [comment, marked as answer]
For sahi+yolov5 please follow this notebook: https://github.com/obss/sahi/blob/main/demo/inference_for_yolov5.ipynb
